### ISSUE TITLE
Code can not be build as of 12/24

### ISSUE THREAD

**Issue by MoJo2600** [awsdeepracer/aws-deepracer-launcher#8]
I'm following the [getting started guide](https://github.com/aws-deepracer/aws-deepracer-launcher/blob/main/getting-started.md) to set up everything on my deepracer.

Sadly i receive an error message when I try to do step 10 of `Building the core packages`. The command `colcon build` throws an error:

`web_video_server_node" links to target "async_web_server_cpp::async_web_server_cpp" but the target was not found.`

I found an [issue ](https://github.com/RobotWebTools/web_video_server/issues/105) in the web_video_server repository stating that support for ROS Foxy has been dropped for the repo. There is a [pull request](https://github.com/RobotWebTools/web_video_server/pull/111) with code to fix the build for foxy and this error goes away when I do checkout the commit `fc7338449da5789a59337d57d7ca374bbd7a0ad9`.

But then another error appears:

```
Starting >>> web_video_server
--- stderr: web_video_server
CMake Error at CMakeLists.txt:6 (find_package):
  By not providing "Findasync_web_server_cpp.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "async_web_server_cpp", but CMake did not find one.

  Could not find a package configuration file provided by
  "async_web_server_cpp" with any of the following names:

    async_web_server_cppConfig.cmake
    async_web_server_cpp-config.cmake

  Add the installation prefix of "async_web_server_cpp" to CMAKE_PREFIX_PATH
  or set "async_web_server_cpp_DIR" to a directory containing one of the
  above files.  If "async_web_server_cpp" provides a separate development
  package or SDK, be sure it has been installed.


---
Failed   <<< web_video_server [1.44s, exited with code 1]

Summary: 0 packages finished [1.87s]
  1 package failed: web_video_server
  1 package had stderr output: web_video_server
```

To summarize: The tutorial is not working anymore and I think it is because that ROS foxy has reached EoL since April 2023 and the referenced packages are not supporting the ROS version anymore.

Do you know if there are any plans to update the ROS version on deepracer anytime in the future?

**Comment by MoJo2600** [comment]
Ok, I digged a little deeper. Compilation succeeds if you do the following:

**Installing build tools > Step 3**

`rosdep update --rosdistro=foxy` 

Otherwise distribution foxy is skipped because it is EoL

And then after **Building the core packages > Step 7** you have to do

`cd ~/deepracer_ws/aws-deepracer-launcher/web_video_server && git checkout fc7338449da5789a59337d57d7ca374bbd7a0ad9`

**Comment by alfrgarc** [comment]
did you finally run the nodes ? because Im having some issues to do it @MoJo2600

**Comment by MoJo2600** [comment]
Yes, I finally got it working. What are your issues? I did the two things mentioned in my commend and could build the code.

**Comment by alfrgarc** [comment]
I'm doing a project in which I have to add some nodes that we have created to the Deepracer nodes and we don't need several nodes that the device incorporates such as inference_pkg, navigation, etc. because we implement our own for autonomous driving but when I run them it gets stuck and tells me this, do you know how I can fix it?

[ctrl_node-2] [INFO] [1738581829.393459817] [ctrl_pkg.ctrl_node]: /model_optimizer_pkg/model_optimizer_server not available, waiting again...
[ctrl_node-2] [INFO] [1738581830.394061878] [ctrl_pkg.ctrl_node]: /model_optimizer_pkg/model_optimizer_server not available, waiting again...
[ctrl_node-2] [INFO] [1738581831.394667288] [ctrl_pkg.ctrl_node]: /model_optimizer_pkg/model_optimizer_server not available, waiting again...

**Comment by alfrgarc** [comment]
I also have a problem with the nodes that I want to add, because it does not detect the launch file but I do not know why, should I add them in the ~/deepracer/aws-deepracer-launcher package or in another package in the ~/deepracer directory?

**Comment by alfrgarc** [comment]
Hello good morning I can't get into the car console and I don't know what the reason is, do I need to install Apache2? 
I have also implemented some nodes to transmit through the manual mode and I do not know where to place the package, I am not interested in inference modes and that, to execute them I only have to comment on the nodes that do not interest me and execute these ?  @MoJo2600

**Comment by alfrgarc** [comment]
To make it only pay attention to manual mode, do you have to do it from the console or not?
